docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: '3.5'
 services:
   timescaledb:
     container_name: timescaledb
-    image: timescale/timescaledb:2.4.1-pg13
+    image: timescale/timescaledb:2.5.0-pg13
```

Comment:
We forgot to update this last time we updated CI, resulting in different behavior for tests. This change fixes the behavior differences.